assets/base.css

```diff
@@ -970,22 +970,23 @@ summary::-webkit-details-marker {
 .grid {
   display: flex;
   flex-wrap: wrap;
-  margin-bottom: 2rem;
-  margin-left: calc(-1 * var(--grid-mobile-horizontal-spacing));;
+  margin-bottom: calc(2rem - var(--grid-mobile-vertical-spacing));
+  margin-left: calc(-1 * var(--grid-mobile-horizontal-spacing));
   padding: 0;
   list-style: none;
 }
 
 @media screen and (min-width: 750px) {
   .grid {
+    margin-bottom: calc(2rem - var(--grid-desktop-vertical-spacing));
```

Comment:
this can create a bit of an overlap but didn't seem to prevent you from clicking the `view all` button: [screenshot](https://screenshot.click/22-01-21ryi-zqhfn.png)